doc/client_lua_api.md

```diff
@@ -683,6 +694,12 @@ Call these functions only at load time!
 * `minetest.after(time, func, ...)`
     * Call the function `func` after `time` seconds, may be fractional
     * Optional: Variable number of arguments that are passed to `func`
+* `minetest.get_us_time()`
+    * Returns time with microsecond precision. May not return wall time.
+* `minetest.get_day_count()`
+    * Returns number days elapsed since world was created, accounting for time changes.
+* `minetest.get_timeofday()`
+    * Returns the time of day: `0` for midnight, `0.5` for midday
```

Comment:
This one is not a complete sentence as there's no personal pronoun. A period would be wrong here and in the most other descriptions. I guess just ignoring this is the easiest way.